src/shared/domain/entities/member.py

```diff
@@ -128,6 +135,9 @@ def validate_email_dev(email) -> bool:
     @staticmethod
     def validate_name(name) -> bool:
 
+        if name == None:
```

Comment:
c ele for none o tipo sera diferente de string

Reply:
ok